autospec/config.py

```diff
@@ -725,6 +731,7 @@ def load_specfile(specfile):
     specfile.extra_make = extra_make
     specfile.extra_make_install = extra_make_install
     specfile.extra_cmake = extra_cmake
+    specfile.cmake_srcdir = cmake_srcdir
```

Comment:
What happens if the `cmake_srcdir` file does not exist so `specfile.cmake_srcdir` gets the current contents of the `cmake_srcdir` variable. Won't the `cmake_srcdir` variable have an empty string and overwrite the default `..`?

Maybe you could check before assignment:
```
specfile.cmake_srcdir = cmake_srcdir or specfile.cmake_srcdir
```
to make sure you aren't overwriting the default with an empty string.

Reply:
sure